cmd/minikube/cmd/start.go

```diff
@@ -173,6 +173,15 @@ func runStart(cmd *cobra.Command, args []string) {
 		os.Setenv(k, v)
 	}
 
+	//Create the host directories if not present
+	hostDirs := []string{viper.GetString(hostConfigDir), viper.GetString(hostDataDir), viper.GetString(hostVolumesDir)}
+	err = cluster.EnsureHostDirectories(libMachineClient, hostDirs)
```

Comment:
On a sidr note, we only need to create the host volume folder, but for completeness we do all.

I agree with the naming. When reviewing just now I had the same problem. It does not read well, as the intention is lost/missing.